packages/suite/src/types/coinmarket/coinmarket.ts

```diff
@@ -266,9 +245,9 @@ export interface CoinmarketGetFiatCurrenciesProps {
 export interface CoinmarketGetCryptoQuoteAmountProps {
     amountInCrypto?: boolean | undefined;
     sendAmount: string;
-    sendCurrency: CryptoSymbol | string | undefined;
+    sendCurrency: CryptoId | string | undefined;
```

Comment:
If `CryptoId` is "more strict" string then we should not allow `string` here (or am I mistaken?)
```suggestion
    sendCurrency: CryptoId | undefined;
```

Reply:
sendCurrency can be also fiat currency